content/css/concepts/colors/terms/rgba/rgba.md

```diff
@@ -51,3 +51,5 @@ h2 {
   color: rgba(0, 0, 0, 25%);
 }
 ```
+
+![Examples 1 and 2](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-colors-rgba-example.png)
```

Comment:
```suggestion
![CSS color rgba example](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-colors-rgba-example.png)
```

The text inside the square brackets acts as alt text. So it should be a phrase about the output image you added, like this:

Reply:
@ishg-153 will do.